packages/@uppy/core/src/index.js

```diff
@@ -1438,15 +1427,7 @@ class Uppy {
     }
 
     // hide the informer after `duration` milliseconds
-    this.infoTimeoutID = setTimeout(this.hideInfo, duration)
-  }
-
-  hideInfo () {
-    const newInfo = { ...this.getState().info, isHidden: true }
-    this.setState({
-      info: newInfo,
-    })
-    this.emit('info-hidden')
+    this.infoTimeoutID = setTimeout(this.#hideInfo, duration)
```

Comment:
i'd just use an arrow function here i guess instead of a property that's only used once, but both is fine